src/MainMenu.cxx

```diff
@@ -42,148 +48,138 @@ bool mainMenu()
     m_AudioMixer.play(AudioTrigger::MainMenu, Coordinate3D{0, 3, 0.5});
 #endif // USE_AUDIO
 
-  Image logo;
-  logo.setTextureID("Cytopia_Logo");
-  logo.setVisibility(true);
-  logo.setPosition(screenWidth / 2 - logo.getUiElementRect().w / 2, screenHeight / 4 - logo.getUiElementRect().h / 2);
-
-  Text versionText;
-  versionText.setText(VERSION);
-  versionText.setPosition(screenWidth - versionText.getUiElementRect().w, screenHeight - versionText.getUiElementRect().h);
-
-  Button newGameButton({screenWidth / 2 - 100, screenHeight / 2 - 20, 200, 40});
-  newGameButton.setText("New Game");
-  newGameButton.setUIElementID("newgame");
-  newGameButton.registerCallbackFunction(
-      []()
-      {
-#ifdef USE_AUDIO
-        playAudioMajorSelection();
-        SignalMediator::instance().signalNewGame.emit(true);
-
-#endif //  USE_AUDIO                                                                                                             \
-    // TODO: Game.run??
-      });
+  auto logoTex = ResourcesManager::instance().getUITexture("Cytopia_Logo");
+  auto iconDiscord = ResourcesManager::instance().getUITexture("Discord_icon");
+  auto iconGithub = ResourcesManager::instance().getUITexture("Github_icon");
+  int logoTexW, logoTexH;
+  SDL_QueryTexture((SDL_Texture*)logoTex, nullptr, nullptr, &logoTexW, &logoTexH);
 
-  Button loadGameButton({screenWidth / 2 - 100, screenHeight / 2 - 20 + newGameButton.getUiElementRect().h * 2, 200, 40});
-  loadGameButton.setText("Load Game");
-  loadGameButton.registerCallbackFunction(
-      []()
-      {
-#ifdef USE_AUDIO
```

Comment:
What happened to playing the sound FX when pressing the button? I don't see any lines of code added that calls the audio.

Reply:
all ok
![image](https://user-images.githubusercontent.com/918081/171583737-6fcf2cae-f1da-4fa3-b224-ef11e6bfca53.png)